libvast/src/system/exporter.cpp

```diff
@@ -42,9 +42,11 @@ namespace system {
 namespace {
 
 void ship_results(stateful_actor<exporter_state>* self) {
-  if (self->state.results.empty() || self->state.stats.requested == 0)
+  VAST_TRACE("");
```

Comment:
What's the benefit of an empty string as trace log entry?

Reply:
You are right to point out that messages like this are a net negative if the log format is not chosen appropriately. The current console logger will just show `ENTER` and `EXIT` messages. But the file logger writes the filename, line and function name, which is mostly what we want from the traces. Also, there is some precedent in the code base for doing it that way.

I would like to extend the logger to support different levels for file and console output. Then the empty trace messages won't be a problem any more.

Reply:
> Also, there is some precedent in the code base for doing it that way.

@Neverlord, would you mind filling in the rationale for those? I think these come from you.

> I would like to extend the logger to support different levels for file and console output. Then the empty trace messages won't be a problem any more.

Makes sense. That seems to be a straight shot in the CAF codebase.

Reply:
This emits valuable information for following program flow with ENTER and EXIT messages for the function. This is used whenever no function arguments are available (other than `self` or the implicit `this` pointer) or are omitted for some other reason (not reasonably printable or something along those lines).